content/collections/audit-works.html

```diff
@@ -1,3 +1,76 @@
+<item>
+  <field name="logo">
+    double.jpeg
+  </field>
+  <field name="title">
+    Double-A-1
+  </field>
+<field name="subtitle">
+  A DeFi primitive for AMM liquidity providing
+  </field>
+  <field name="date">
+    2023-01-20
+  </field>
+  <field name="about">
+
+  </field>
+  <field name="report">
+    /library/audits/double-1
+  </field>
+  <field name="attachments">
+    [GitHub Organization](https://github.com/double2win) <i>- Private Repo</i>
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    citadel.jpg
+  </field>
+  <field name="title">
+    Citadel-A-3
+  </field>
+  <field name="subtitle">
+    A fully on-chain game
+  </field>
+  <field name="date">
+    2023-06-26
+  </field>
+  <field name="about">
+
+  </field>
+  <field name="report">
+    /library/audits/citadel-3
+  </field>
+  <field name="attachments">
+    [Citadel Game Docs](https://library.citadel.game/)
+    [GitHub Organization](https://github.com/thecitadelgame/) - <i>Private Repo</i>
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    nori.png
+  </field>
+  <field name="title">
+    Nori-4
+  </field>
+  <field name="subtitle">
+    Carbon removal marketplace
+  </field>
+  <field name="date">
+    2023-10-02
+  </field>
+  <field name="about">
```

Comment:
Finished the second part of auditing core Nori contracts such as Market, Certificate, Removal, RestrictedNORI, and library helpers